test/language/literals/regexp/S7.8.5_A1.2_T2.js

```diff
@@ -2,15 +2,27 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-info: "RegularExpressionFirstChar :: * or \\ or / or [empty] is incorrect"
-es5id: 7.8.5_A1.2_T2
-description: \
+esid: sec-literals-regular-expression-literals
+info: |
+  RegularExpressionBody ::
+    RegularExpressionFirstChar RegularExpressionChars
+
+  RegularExpressionChars ::
+    [empty]
+    RegularExpressionChars RegularExpressionChar
+
+  RegularExpressionFirstChar ::
+    RegularExpressionNonTerminator but not one of * or \ or / or [
+
+  RegularExpressionNonTerminator ::
+    SourceCharacterbut not LineTerminator
+
+description: RegularExpressionFirstChar :: RegularExpressionNonTerminator but not one of * or \ or / or [, \ case
```

Comment:
yaml parse is failing, I guess  you might need to scape this in quotes or using a `description: > \n`  format

Reply:
Whoops!

Reply:
Fixed